libraries/DNSServer/src/DNSServer.h

```diff
@@ -1,5 +1,7 @@
 #ifndef DNSServer_h
 #define DNSServer_h
+
+#include <memory>
```

Comment:
future-proofing, I tried adding DNSServer/examples/DNSServer to host test script, but I see valgrind error in UDP code :shrug:
something to note